docs/ethereum_attestation.md

```diff
@@ -0,0 +1,133 @@
+# RFC: Ethereum attestation
+
+* Author: @CodeSandwich
+* Date: 2021-03-05
+* Status: draft
+* Community discussion: n/a
+
+## Motivation
+
+The attestation between Link and Ethereum is a valuable building block for a user identity.
+It brings the Link reputation coming from projects and contributions to the
+Ethereum world of DAOs and donations, where it's important to know who's behind an address.
+On the other hand, it lends Ethereum account reputation with its assets and undeniable history
+to Link to build user's trust in the identity.
+
+## Overview
+
+This RFC is built on top of the [identities spec][identities].
+It introduces support for Ethereum address claims on Link
+and a smart contract on Ethereum to make Link identity claims on Ethereum.
+
+## Link identity JSON extension
+
+The identity doc `payload` structure supports a new key: `https://radicle.xyz/ethereum/claim/v1`.
+It can be used only in person identities.
+Under this key there is stored an Ethereum address claim in this format:
+
+- `address` - the claimed Ethereum address, encoded according to [EIP-55][eip-55],
+e.g. using [ethers.js][ethers-addr]
+- `expiration` - the claim expiration timestamp, encoded as a [JavaScript Date][date].
```

Comment:
> `expiration` - the claim expiration timestamp, encoded as a [JavaScript Date][date].

Since we have changed the type of the `expiration` date, I just want to make sure this is still correct as I am afraid it's not.

Reply:
I think that it's still valid, we've just added verification of that format.